Frontend/ui-driver/src/Strings/TA.js

```diff
@@ -304,6 +304,7 @@ const tamilStrings = {
   CUSTOMER_NOTIFIED: "வாடிக்கையாளருக்கு அறிவிக்கப்பட்டது",
   I_ARRIVED: "வந்துவிட்டேன்",
   ESTIMATED_RIDE_FARE: "மதிப்பிடப்பட்ட சவாரி கட்டணம்: ",
+  RIDE_FARE : "சவாரி கட்டணம்",
```

Comment:
is it added for other languages as well ?

Reply:
yes it was not included just for this language
<img width="331" alt="Screenshot 2023-09-18 at 8 03 12 PM" src="https://github.com/nammayatri/nammayatri/assets/55204010/c752dc29-32bc-4ab7-822f-5f44a214ee4b">